### PR TITLE
DevWs for frient containing containing SmartSense Moisture Sensor

### PR DESCRIPTION
**Pull request by frient-design**

### CODE REVIEW

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy [hunk 1]
```diff
@@ -1,279 +1,296 @@
...
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "CentraLite", model: "3315-G", deviceJoinName: "Centralite Water Leak Sensor" //Centralite Water Sensor
+		fingerprint inClusters: "0000,0001,0003,000F,0020,0402,0500", outClusters: "0019", manufacturer: "SmartThings", model: "moisturev4", deviceJoinName: "Water Leak Sensor", mnmn: "SmartThings", vid: "smartthings-water-leak-3315S-STSWTR"
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500", outClusters: "0019", manufacturer: "Samjin", model: "water", deviceJoinName: "Water Leak Sensor", mnmn: "SmartThings", vid: "smartthings-water-leak-IM6001"
+		fingerprint inClusters: "0000,0001,0003,0020,0402,0500,0B05", outClusters: "0019", manufacturer: "Sercomm Corp.", model: "SZ-WTD03", deviceJoinName: "Sercomm Water Leak Sensor" //Sercomm Water Leak Detector
+        fingerprint profileId: "0104", deviceId: "0402", inClusters: "0000,0001,0003,000F,0020,0500,0502", outClusters: "000A,0019", manufacturer: "frient A/S", model :"FLSZB-110", deviceJoinName: "frient Water Leak Detector"
```
Comment (PKacprowiczS) [inline devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy]: Please change deviceJoinName to `frient Water Leak Sensor`. Also please change spaces to tabs.

**Review by PKacprowiczS — CHANGES_REQUESTED** [review]
please adjust code in your request so diff here in PR would get fixed (I'm not sure what might be wrong, maybe it's encoding)

File: devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy (hunk 1 above)
Comment (rboy1) [reply devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy]: fixed

**Comment by frient-design** [comment]
We have made all the requested changes, can you please accept the request to continue the certification or let us know if more modification is needed

**Comment by PKacprowiczS** [comment]
@frient-design Those PRs which still have issue with wrong diff won't be accepted.
https://docs.github.com/en/free-pro-team@latest/github/using-git/configuring-git-to-handle-line-endings maybe you'll find something useful at this link

**Review by rboy1 — DISMISSED** [review]
Fixed end of line encoding issues

**Comment by rboy1** [comment]
@PKacprowiczS should be good now

**Comment by PKacprowiczS** [comment]
@rboy1 I think you should implement same change as was discussed here https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/49364#pullrequestreview-531606161 
See https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/50163 for reference

**Review by rboy1 — APPROVED** [review]
Inlined endpoint and used constant for battery attributes

**Comment by rboy1** [comment]
@PKacprowiczS fixed, also changed the battery attributes to constants as suggested by @dkirker https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/49599#discussion_r524512446

**Review by PKacprowiczS — APPROVED** [review]
Looks good
@greens @dkirker @KKlimczukS

**Review by greens — APPROVED** [review]

**Pull request merged**